GiEnJul/Entities/Person.cs

```diff
@@ -7,11 +9,8 @@ public Person()
 
         // RowKey = Guid
         // PartitionKey = {recipientId}
-        public Person(string recipientId, string rowKey) : base(recipientId, rowKey)
-        {
-            PartitionKey = $"{recipientId}";
-            RowKey = rowKey;
-        }
+        public Person(string recipientId) : base(recipientId ?? throw new ArgumentNullException(nameof(recipientId)), Guid.NewGuid().ToString())
```

Comment:
```
recipientId ?? throw new ArgumentNullException(nameof(recipientId))
```

Do this inside the constructor of BaseEntity?

Reply:
Good idea 👍